src/host/getset.cpp

```diff
@@ -599,15 +599,14 @@ void ApiRoutines::GetLargestConsoleWindowSizeImpl(const SCREEN_INFORMATION& cont
         auto& gci = g.getConsoleInformation();
 
         const auto coordScreenBufferSize = context.GetBufferSize().Dimensions();
-        const auto requestedBufferSize = data.dwSize;
-        if (requestedBufferSize.X != coordScreenBufferSize.X ||
-            requestedBufferSize.Y != coordScreenBufferSize.Y)
+        const auto requestedBufferSize = til::wrap_coord_size(data.dwSize);
+        if (requestedBufferSize != coordScreenBufferSize)
         {
             auto& commandLine = CommandLine::Instance();
 
             commandLine.Hide(FALSE);
 
-            LOG_IF_FAILED(context.ResizeScreenBuffer(til::wrap_coord_size(data.dwSize), TRUE));
+            LOG_IF_FAILED(context.ResizeScreenBuffer(requestedBufferSize, TRUE));

```

Comment:
Note to other reviewers: aside from removing the union in til::point/rect/size, this is the only other change I found that isn't just renaming.